fiftyone/migrations/revisions/v0_20_0.py

```diff
@@ -0,0 +1,61 @@
+"""
+FiftyOne v0.20.0 revision.
+
+| Copyright 2017-2023, Voxel51, Inc.
+| `voxel51.com <https://voxel51.com/>`_
+|
+"""
+
+
+def up(db, dataset_name):
```

Comment:
Is this going to be a slow upgrade process for large datasets? I have a limited concept of how fast adding a field to all samples and frame samples would be

Reply:
Hand-wavy guess: on the order of a few seconds per million documents being updated. Depends on database configuration, load, etc of course.

It's definitely a more expensive operation